server/index.js

```diff
@@ -61,33 +62,24 @@ app.get('/legacy/:id', (req, res) => {
 // Create, Read, Update, Delete
 
 // Create
-app.post('/v2/house/:id/:check_in_date/:check_out_date/:adults/:childs/:infants', (req, res) => {
-  res.send(req.params)
-})
+app.post('/v2/houses/:id/reservations/:check_in_date/:check_out_date/:adults/:children/:infants', (req, res) => {
```

Comment:
I think it's ok to have :reservation_id as a param, but for the check_in_date, check_out_date, adults, children, infants may be better to have as query elements.

i.e.
/v2/houses/:id/reservations/:reservation_id?check_in_date=123456&check_out_date=345678&...

However, I'm not sure, so check what conforms to REST standards.